freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -81,7 +81,6 @@ def handle(self, event):
             msg = f"The are no digests for NVRs: {','.join(original_nvrs)}"
             log.warning(msg)
             db_event.transition(EventState.SKIPPED, msg)
-            db.session.commit()
```

Comment:
I think this is still needed unless the `transition` method calls this.

Reply:
That was actually the reason why I deleted it, `transition` calls it before publishing messages and returning.